### ISSUE TITLE
Wiki: Partial DB flow

### ISSUE THREAD

**Issue by sorja** [openforis/fra-platform#4164]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by minotogna** [comment]
@sorja do we need this task?

**Comment by sorja** [comment]
> [@sorja](https://github.com/sorja) do we need this task?

Yes, looking for a short moment where I can add some screenshots and write the flow
Task is more a reminder for me

**Comment by sorja** [comment]
Heroku DB so small as this is not needed, should delete the workflow for partial db and update the heroku db workflow to upload to slack
